README.md

```diff
@@ -1,32 +1,37 @@
 # 非 k8s 环境下部署监控使用文档
 
-## 编写 config.yaml 文件
+### 配置nebula-stats-exporter
 
-在 `config.yaml` 中的 `nebulaItems` 下添加监控的 nebula 组件，`instanceName` 用于标示组件名称，`endpointIP` 用于指定该组件的 IP 地址，`endpointPort` 用于指定该组件的 http 端口，`nebulaType`用于指定该组件的服务类型: `metad`、`storaged` 或者 `graphd`。
-例子如下：
+在config.yaml中的nebulaItems下添加监控的nebula组件，instanceName用于标识组件名称，endpointIP用于指定该组件的 IP 地址，
+endpointPort用于指定该组件的 http 端口，nebulaType用于指定该组件的服务类型: metad、storaged 或者 graphd.  
+如果你是rpm方式部署nebula，那么endpointIP请填写主机IP地址，如果你是docker方式部署nebula，那么endpointIP请填写容器IP地址.
+  
+配置示例文件可参考deploy/bare-metal/config.yaml
 
 ```yaml
 nebulaItems:
   - instanceName: metad-0
-    endpointIP: 127.0.0.1 # nebula host IP
+    endpointIP: 192.168.8.102 # nebula component ip address
```

Comment:
Why change the ip?

Reply:
The address 127.0.0.1 may confuse the user, so change to a relatively real address